package.json

```diff
@@ -74,7 +75,8 @@
   },
   "remarkConfig": {
     "plugins": [
-      "remark-preset-lint-recommended"
+      "remark-preset-lint-recommended",
+      "remark-lint-link-title-style"
```

Comment:
I gave it a spin by setting one title to use single quotes and then running `npm run lint`. It did not throw an error, thus the plugin does not work for frontmatter.